app/contracts/groups/delete_contract.rb

```diff
@@ -28,6 +28,6 @@
 
 module Groups
   class DeleteContract < ::DeleteContract
-    delete_permission -> { user.active? && user.admin? }
+    delete_permission(:admin)
```

Comment:
What is the reason for this change? I can't say that it is wrong but it seems to deliberate to be checking for whether the user is active that I would surmise that part being the result of a bug fix.

Reply:
My understanding is that there is no change in behavior.
If we check `DeleteContract` class. Then using `delete_permission(:admin)` here will force `authorized?` to fall into [this line](https://github.com/opf/openproject/blob/dev/app/contracts/delete_contract.rb#L59) which has the same logic.